com.unity.mobile.notifications/Runtime/iOS/iOSNotification.cs

```diff
@@ -73,8 +74,7 @@ public enum NotificationSoundType
 
     /// <summary>
     /// Importance and delivery timing of a notification.
-    /// See Apple documentation for details. Available since iOS 15, always Active on lower versions.
-    /// <see href="https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel?language=objc"/>
+    /// See <a href="https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel">Apple documentation</a> for details. Available since iOS 15, always Active on lower versions.
```

Comment:
```suggestion
    /// For more information, refer to <a href="https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel">Apple documentation</a>. UNNotificationInterruptionLevel is available since iOS version 15. On versions earlier than iOS 15, the interruption level is always active.
```